spellbook/models/opensea/ethereum/opensea_v1_ethereum_fees.sql

```diff
@@ -0,0 +1,30 @@
+
+ {{ config(schema = 'opensea_v1_ethereum', 
+alias='fees') }}
+
+SELECT  
+    CASE WHEN size(trace_address) = 1 then array(3::bigint) -- for single row join
+    WHEN size(trace_address) = 2 then array(trace_address[0]) 
+    WHEN size(trace_address) = 3 then array(trace_address[0], trace_address[1])
+    END as trace_address,
+    tx_hash,
+    SUM(value) AS fees,
+    to,
+    'ETH' as fee_currency_symbol
+FROM ethereum.traces source_fees
+WHERE 
+FROM IN ('0x7be8076f4ea4a4ad08075c2508e481d6c946d12b','0x7f268357a8c2552623316e2562d90e642bb538e5')
+AND to = '0x5b3256965e7c3cf26e11fcaf296dfc8807c01073' -- OpenSea Wallet
```

Comment:
would be cool to make this a public variable but that doesn't need to be solved in this pr